### DISCUSSION TITLE
Использование в NUXT

### DISCUSSION THREAD

**Discussion by ValievE** [yandex-maps-unofficial/vue-yandex-maps#299] [Q&A (1.x)] [answered]
Добрый день еще раз. Сегодня писал про проблемы в NUXT, но даже перейдя на нужную версию (npm install vue-yandex-maps@~0.12, ставится версия 0.12.7) все равно есть проблемки. Думаю стоит начать, что документация с нового репозитория открывается старая, с npm install vue-yandex-maps и не совсем понятно как настроить плагин под NUXT

**Comment by daniluk4000** [comment]
Здравствуйте, в новом репозитории документация актуальная. Верните версию 2.1.2 и сделайте по документации из этого репозитория (и в той ишью я вам кидал ссылку на гайд по установке)

**Reply by daniluk4000** [reply, marked as answer]
Базово проблем возникать не должно, кроме первой ссылки в гугле.

- Установить последнюю (2.1.2) vue-yandex-maps
- Добавить `vue-yandex-maps/nuxt` в modules
- Заполнить apikey в `yandexMaps`

Готово. Всё как тут: https://yandex-maps-unofficial.github.io/vue-yandex-maps/guide/vue3.html#nuxt-3

**Reply by animus1385** [reply]
Я делаю все тоже самое но у меня возникает это.
![Screenshot_17](https://github.com/user-attachments/assets/16d39cb3-684a-4569-9a8a-effb27384a7f)


делал все по инструкции
![Screenshot_16](https://github.com/user-attachments/assets/fd4c48da-f128-4baa-b391-9414a627ddbc)
![Screenshot_15](https://github.com/user-attachments/assets/9bf38f0c-8d42-43ee-808f-8b868ec86e75)

**Reply by daniluk4000** [reply]
Мне это похоже уже надо красным текстом в шапке доки и репы писать.

Проверьте Referer в настройках ЛК яндекса

![image](https://github.com/user-attachments/assets/48a35d6f-423a-4ff4-90cb-bef8d029401d)
